### ISSUE TITLE
Wells without curves

### ISSUE THREAD

**Issue by dworley33** [agile-geoscience/welly#89]
I am having  problems when trying to export my project to a matrix. I am getting a 'NoneType' error due to some of the wells in project do not have a curve which I need to export. How can I fix this issue so I can export as a matrix?

**Comment by dworley33** [comment]
Yeah i have been following the welly to the wescue workflow and here is the step I am getting an error on.
![line of code](https://user-images.githubusercontent.com/42754731/44819352-2b131780-abb2-11e8-8f3e-d0afb8729047.PNG)

**Comment by dworley33** [comment]
here is the beginning of my code
![image](https://user-images.githubusercontent.com/42754731/44819469-95c45300-abb2-11e8-82d9-1f4d1b7b1f38.png)

**Comment by EvanBianco** [comment]
@dworley, perhaps we need to fill the wells without PHIN with Curves filled with Nans so a 2-d array can be created. I dont think we have dealt with this use case before so please bear with us as we develop a solution to handle this. Suggestions welcome.

**Comment by EvanBianco** [comment]
@dworley,
do you get the expected behavior if 

`keys = ['GR', 'RESD', 'RHOB', 'FACIES']`

**Comment by dworley33** [comment]
I am a beginner with python but from my research I gather online it looked like nans could possibly work for this issue. I have tried it without PHIN in the keys and the same error occurs. Some of our GR values are also -999. So that might be the reason for the error using the KEYS without PHIN.
